configuration/introduction.md

```diff
@@ -93,15 +143,103 @@ MY_HTTPS=false
 
 ##### Using linux environment variables
 
-You have to declare them before you start MagicMirror², e.g. by executing:
+define them before you start MagicMirror², in a bash script, for example: 
 
-```shell
+```bash
+cd ~/MagicMirror
 export MY_ADDRESS=localhost
 export MYPORT=8080
 export MY_HTTPS=false
+npm start
+```
+
+
+
+#### Using `electronOptions`: 
+
+The most common use for `electronOptions` is a dual monitor setup on a Raspberry Pi 4 or greater. To do so, you can launch a second MagicMirror and use `electronOptions` to move it over to the next monitor. For example: 
+
+```bash
+var config = {
+	electronOptions: { x: 1920 },
+	...
+```
+
+
+
+#### Real World Examples: 
+
+##### Two Screens: 
+
+A user has two monitors running on a Pi4B.  User wants to show cameras on one monitor, and wants to show informational panels on the other. Both Monitors are set up to 1920x1080 resolution, and within the Pi's OS, they're virtually located so left monitor is directly left of the right monitor, creating a effective 3840x1080 screen. 
+
+User would have to set up two config files, one custom.css, and two starting scripts to do so. User would use method of choice to launch starting scripts (e.g.: `PM2`). 
+
+Starting Script 1 (EG: mm.sh): 
+
+```bash
+cd ~/MagicMirror
+npm start
+```
+
+Starting Script 2 (EG: mm2.sh): 
+
+```bash
+cd ~/MagicMirror
+export MM_CONFIG_FILE=${pwd}/config/config2.js
```

Comment:
export MM_CONFIG_FILE=$(pwd)/config/config2.js

Reply:
The problem is that parens don't seem to work on Pi/bookworm. 

```key1970@officemirror:~/MagicMirror $ export MM_CONFIG_FILE=$(pwd)/config/config2.js
bkey1970@officemirror:~/MagicMirror $ export MM_PORT=8081
bkey1970@officemirror:~/MagicMirror $ DISPLAY=:0 npm start

> magicmirror@2.25.0 start
> DISPLAY="${DISPLAY:=:0}" ./node_modules/.bin/electron js/electron.js

[26.12.2023 14:56.06.740] [LOG]   Starting MagicMirror: v2.25.0
[26.12.2023 14:56.06.749] [LOG]   Loading config ...
[26.12.2023 14:56.06.752] [DEBUG] config template file not exists, no envsubst
[26.12.2023 14:56.06.758] [LOG]   Loading module helpers ...
[26.12.2023 14:56.06.767] [LOG]   Initializing new module helper ...
[26.12.2023 14:56.06.769] [LOG]   Module helper loaded: MMM-EmbedURL
[26.12.2023 14:56.06.769] [LOG]   All module helpers loaded.
[26.12.2023 14:56.06.781] [LOG]   Starting server on port 8081 ...
[26.12.2023 14:56.06.787] [WARN]  You're using a full whitelist configuration to allow for all IPs
[26.12.2023 14:56.06.825] [LOG]   Server started ...
[26.12.2023 14:56.06.827] [LOG]   Connecting socket for: MMM-EmbedURL
[26.12.2023 14:56.06.828] [LOG]   Sockets connected & modules started ...
[26.12.2023 14:56.07.136] [LOG]   Launching application.
```

Results in the defaults.js config taking over and giving me "No Config".